Add city names to extras.xliff

brizental opened the pull request:
I commited a mistake in my last PR and forgot to include city names for translation.

I understand this is a bit annyoing because we have already some translations for the new strings.

I have kept that in mind and attempted not to remove any of the new translations.

flodolo commented on it/extras.xliff:
```diff
@@ -166,162 +166,450 @@
       <trans-unit id="servers.us" xml:space="preserve">
         <source>United States of America</source>
-        <target>Stati Uniti d’America</target>
+        <target>Stati Uniti d'America</target>
```
```suggestion
        <target>Stati Uniti d’America</target>
```

This is reverting a change, but it seems isolated.

flodolo commented on fy_NL/extras.xliff:
```diff
@@ -166,162 +166,570 @@
         <source>Albania</source>
         <target>Albaanje</target>
       </trans-unit>
+<<<<<<< HEAD
```
Completely missed these rebase failures :-(

flodolo commented on hu/extras.xliff:
```diff
@@ -166,162 +166,450 @@
+        <source>Johannesburg</source>
+        <target></target>
+      </trans-unit>
+    </body xml:space="preserve">
```
Something went wrong at the end of this file.

flodolo reviewed (commented):
Found a few things to issue, which I should have caught earlier (didn't try to parse the XLIFF) :-(

brizental commented:
>  Found a few things to issue, which I should have caught earlier (didn't try to parse the XLIFF) :-(

Sheesh, really sorry about that. Should have done that myself as well. I see you have already addressed the issues, thanks a lot!
